### PR TITLE
BAEL-4224: Hidden inputs with thymeleaf

### PR DESCRIPTION
**Pull request by michael-pratt**

### CODE REVIEW

File: spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogDTO.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package com.baeldung.thymeleaf.blog;
```
Comment (rpvilao) [inline spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogDTO.java:1]: just a thought, could we use lombok here?

Comment (michael-pratt) [reply spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogDTO.java:1]: We could definitely use it here, but it wasn't already imported the module and I wanted to keep the example code as much aligned to the article as possible.

File: spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogController.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.Random;
+
+@Controller
+public class BlogController {
+
+    @GetMapping("/blog/new")
+    public String newBlogPost(Model model)
+    {
```
Comment (KevinGilmore) [inline spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogController.java:14]: As per guidelines, the opening brace should be at the end of line 13, not by itself.

Comment (michael-pratt) [reply spring-thymeleaf-3/src/main/java/com/baeldung/thymeleaf/blog/BlogController.java]: Moved to same line

**Pull request merged**